pandas_profiling/__init__.py

```diff
@@ -1,153 +1,153 @@
-# -*- coding: utf-8 -*-
 """Main module of pandas-profiling.
 
-Docstring is compliant with NumPy/SciPy documentation standard:
-https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt
-See also for a short description of docstring:
-https://stackoverflow.com/questions/3898572/what-is-the-standard-python-docstring-format
+.. include:: ../README.md
```

Comment:
what does this means ?:) is this on purpuse

Reply:
This is markup (restructuredtext) to include the README in the documentation!

Reply:
cool never knew you can inject markdown via .. include: prefix